### PR TITLE
More tests for Mat

### PR DESCRIPTION
**Pull request by kshyatt**
Added some tests for `resize` error throws, `lengthlocal`, `conj`, negating a matrix, and `./` and `.\` for `Number`s.

### CODE REVIEW

**Comment by jakebolewski** [comment]
Hmm, needs a rebase now.

**Comment by kshyatt** [comment]
Will do

File: test/mat.jl [hunk 1]
```diff
@@ -338,4 +344,30 @@ for i=1:3
   end
 end
 
+mat22 = mat16/2
+mat22j = mat16j/2
+
+for i=1:3
+  for j=1:3
+    @fact mat22[i,j] => roughly(mat22j[i,j])
```
Comment (jakebolewski) [inline test/mat.jl]: Also, try to use the new `-->` factcheck syntax

**Comment by kshyatt** [comment]
Yeah I'll fix that in the rebase

**Comment by kshyatt** [comment]
More conflicts? :/

Well I found and fixed another bug.

**Comment by jakebolewski** [comment]
:+1:

**Pull request merged**